.coveragerc

```diff
@@ -286,6 +286,7 @@ omit =
     homeassistant/components/hydrawise/*
     homeassistant/components/hyperion/light.py
     homeassistant/components/ialarm/alarm_control_panel.py
+    homeassistant/components/iaqualink/*
```

Comment:
The config flow module requires 100 % test coverage so don't exclude that module from coverage calculation.

Reply:
I can't seem to find any config_flow module with test coverage in the dev branch. What am I missing?

Reply:
See eg hue config flow tests:
https://github.com/home-assistant/home-assistant/blob/dev/tests/components/hue/test_config_flow.py

Reply:
Ha, I looked for files called test* under homeassistant/components. That would explain why I couldn't find anything. Thanks.